include/openssl/ssl.h

```diff
@@ -282,6 +282,9 @@ typedef int (*SSL_custom_ext_parse_cb_ex) (SSL *s, unsigned int ext_type,
 /* Typedef for verification callback */
 typedef int (*SSL_verify_cb)(int preverify_ok, X509_STORE_CTX *x509_ctx);
 
+/* In TLSv1.3 allow a non-(ec)dhe based kex_mode */
+# define SSL_OP_ALLOW_NO_DHE_KEX                         0x00000001U
```

Comment:
This is the first re-use of an SSL_OP value since these have been deprecated.
As of 1.1.0e, SSL_OP_ALL (0x80000BFFU) included this bit in it's value. If 1.1.0 and 1.1.1 are supposed to be binary compatible, will this cause any problems for those compiled against 1.1.0e (or earlier) and using SSL_OP_ALL ?